File: 100-Days-Series/Day 95/SumOfPositiveNumbers.js

// Function to find the sum of positive numbers in the array.
const getSumOfPositiveNumbers=(arr)=>{
    let sum=0;
    for(let i=0;i<arr.length;i++){
        if(arr[i]>0){
            sum+=arr[i];
        }
    }
    return sum;
}
// Let's Start Here
const arr=[-2, 3, 4, -1, 5];
let result=getSumOfPositiveNumbers(arr);
console.log("Sum of Positive Numbers : "+result);

File: 100-Days-Series/Day 95/SumOfPositiveNumbers.cpp
#include<iostream>
#include<vector>
using namespace std;
// Function to find the sum of positive numbers in the array.
int getSumOfPositiveNumbers(vector<int>&arr){
    int sum=0;
    for(int i=0;i<arr.size();i++){
        if(arr.at(i)>0){
            sum+=arr.at(i);
        }
    }
    return sum;
}
// Let's Start Here
int main(){
    vector<int>arr={-2, 3, 4, -1, 5};
    int result=getSumOfPositiveNumbers(arr);
    cout<<"\nSum of Positive Number : "<<result<<endl;
    return 0;
}